js/ui/grid_core/ui.grid_core.data_source_adapter.js

```diff
@@ -1,899 +1,2 @@
-            return !!this._isCustomLoading;
-        },
-        load: function(options) {
-            const that = this;
-            const dataSource = that._dataSource;
-            // @ts-expect-error
-            const d = new Deferred();
-
-            if(options) {
-                const store = dataSource.store();
-                const dataSourceLoadOptions = dataSource.loadOptions();
-                const loadResult = {
-                    storeLoadOptions: extend({}, options, { langParams: dataSourceLoadOptions?.langParams }),
-                    isCustomLoading: true
-                };
-
-                // @ts-expect-error
-                each(store._customLoadOptions() || [], function(_, optionName) {
-                    if(!(optionName in loadResult.storeLoadOptions)) {
-                        loadResult.storeLoadOptions[optionName] = dataSourceLoadOptions[optionName];
-                    }
-                });
-
-                this._isLoadingAll = options.isLoadingAll;
-
-                that._scheduleCustomLoadCallbacks(d);
-                // @ts-expect-error
-                dataSource._scheduleLoadCallbacks(d);
-
-                that._handleCustomizeStoreLoadOptions(loadResult);
-                executeTask(function() {
-                    if(!dataSource.store()) {
-                        return d.reject('canceled');
-                    }
-
-                    when(loadResult.data || that.loadFromStore(loadResult.storeLoadOptions)).done(function(data, extra) {
-                        loadResult.data = data;
-                        loadResult.extra = extra || {};
-                        that._handleDataLoaded(loadResult);
-
-                        if(options.requireTotalCount && loadResult.extra.totalCount === undefined) {
-                            loadResult.extra.totalCount = store.totalCount(loadResult.storeLoadOptions);
-                        }
-                        // TODO map function??
-                        when(loadResult.data, loadResult.extra.totalCount).done(function(data, totalCount) {
-                            loadResult.extra.totalCount = totalCount;
-                            d.resolve(data, loadResult.extra);
-                        }).fail(d.reject);
-                    }).fail(d.reject);
-                }, that.option('loadingTimeout'));
-
-                return d.fail(function() {
-                    that._eventsStrategy.fireEvent('loadError', arguments);
-                }).always(() => {
-                    this._isLoadingAll = false;
-                }).promise();
-            } else {
-                return dataSource.load();
-            }
-        },
-        reload: function(full) {
-            return full ? this._dataSource.reload() : this._dataSource.load();
-        },
-        getCachedStoreData: function() {
-            return this._cachedStoreData;
-        }
-    };
-
-    return members;
-})());
+/** @deprecated */
```

Comment:
Let's change the js-doc comment to the usual comment
```javascript
/** @deprecated */
```
to
```javascript
// deprected
```

Because js-doc comments can cause some strange behaviour in our metadata-tools :(